gh-md-toc

```diff
@@ -225,6 +231,7 @@ gh_toc_app() {
         echo ""
         echo "Usage:"
         echo "  $app_name [--insert] src [src]  Create TOC for a README file (url or local path)"
+	echo "  $app_name [--ins-nb] src [src]  Create TOC without backup, requires <!--ts--> / <!--te--> placeholders"
```

Comment:
1. let's use spaces instead of tabs
2. in the usage section there should be a real options. i don't see such option `--ins-nb` later